django/chat/models.py

```diff
@@ -281,7 +288,7 @@ def sorted_files(self):
 
     @property
     def sources(self):
-        return self.answersource_set.all().order_by("-node_score")
+        return self.answersource_set.all()
```

Comment:
Removing the ordering here lets their display order be dictated by chat_options.